cms/models.py

```diff
@@ -1071,6 +1113,149 @@ def get_context(self, request, *args, **kwargs):
         return context
 
 
+class SignatoryPage(Page):
+    """ CMS page representing a Signatory. """
+
+    promote_panels = []
+    parent_page_types = [SignatoryIndexPage]
+    subpage_types = []
+
+    name = models.CharField(
+        max_length=250, null=False, blank=False, help_text="Name of the signatory."
+    )
+    title_1 = models.CharField(
+        max_length=250,
+        null=True,
+        blank=True,
+        help_text="Specify signatory first title in organization.",
+    )
+    title_2 = models.CharField(
+        max_length=250,
+        null=True,
+        blank=True,
+        help_text="Specify signatory second title in organization.",
+    )
+    organization = models.CharField(
+        max_length=250,
+        null=True,
+        blank=True,
+        help_text="Specify the organization of signatory.",
+    )
+
+    image = models.ForeignKey(
```

Comment:
Can we just go ahead and rename this to `signature_image`? It will remove any chance of a confusion.

Reply:
sure.